### DISCUSSION TITLE
Line Highlight

### DISCUSSION THREAD

**Discussion by boocha34** [semiautomaticgit/SemiAutomaticClassificationPlugin#192] [Ideas]
hello my name is Thabo Maqelepo from Lesotho, I am currently using the SCP plugin for land cover, and i was wondering if it could be possible to add a "Line ROI" feature along with the polygon and pointer. this could be helpful in a case where the features are thin of about 1 pixel thickness as is the case with the roads and rivers I am dealing with using the Landsat image (170-080)
![training](https://user-images.githubusercontent.com/18643647/139637729-61be8868-6668-4170-ba72-369bc2a36965.JPG)
.

**Comment by semiautomaticgit** [comment]
Hello Thabo Maqelepo, thank you for your suggestion.
I'll try to add this function in a future version of the plugin.
At the moment you could use QGIS vector tools to create a line vector, create the lines for the ROIs, then create a buffer of half the pixel size, the import the buffered vector in SCP using this tool https://semiautomaticclassificationmanual.readthedocs.io/en/latest/importSignaturesTab.html
